app/views/shared/_analysis_controls.html.erb

```diff
@@ -17,20 +17,20 @@
   <div class='analysis-controls' style='display: none;'>
     <% if local_assigns[:analysis_controls] %>
       <div class="d-flex align-items-center justify-content-between" role="group">
-        <span class="mr-2 align-bottom">Explore</span>
+        <span class="mr-2 align-bottom"><%= t('analysis_controls.explore') %></span>
         <div>
           <button class="btn btn-sm btn-outline-dark rounded-pill explore-button move_back" disabled>
-            <%= fa_icon('backward') %> Back 1 <span class='period'>period</span>
+            <%= fa_icon('backward') %> <%= t('analysis_controls.back_one') %> <span class='period'>period</span>
```

Comment:
I think the "period" default label for back, up, forward should also be translated?

Reply:
:+1: